Description: Bug field now contains default value, it should be changed as actual upstream url
Author: Hideki Yamane <henrich@debian.org>
Forwarded: Yes
Bug: <url in upstream bugtracker>

--- a/foobar.h
+++ b/foobar.h
@@ -20,6 +20,6 @@
 
 
 /**
- * You may know.
+ * You may know...
  */
 
